### ISSUE TITLE
-'NA' bullets added to each entry

### ISSUE THREAD

**Issue by will-ball** [nstrayer/datadrivencv#19]
Having lots of fun creating my CV but I've just run into an issue where an extra bullet point has been added to my entries when I render. It seems to be taking an empty 'description' column as NA for each item.

**Comment by will-ball** [comment]
I've just had a look through the CV_printing_functions.R file (which I previously haven't touched) and it seems like these NAs should be filtered out but something must be going wrong as all of my entries have them.

**Comment by nstrayer** [comment]
Hmm. Can you paste a screenshot? 

Maybe you put an "NA" in your positions data somewhere? Pagedown ignores things that are printing as "N/A" whereas R missing as `NA`. This means I have to convert between the two but it's possible that I missed a conversion somewhere and so pagedown's getting the raw R `NA` instead of the one it needs.

**Comment by will-ball** [comment]
Hi Nick,

Here are some screenshots. One of the .html once it's rendered and one of the googlesheet with the empty cells.
![gdna](https://user-images.githubusercontent.com/61555378/83239795-5dd7ae80-a190-11ea-93b8-a3b151bf3eac.png)
![cvna](https://user-images.githubusercontent.com/61555378/83239805-616b3580-a190-11ea-838b-c1e3828f3bec.png)

One of the entries doesn't have a description at all - I tried filling that in just in case it was causing an issue but still got the same.

**Comment by MLiberati** [comment]
Hi, Nick! I'm having a similar issue, just without the 'NA' text added. Bullet-points are being added below entries even if I have nothing in the description cells. My source files are CSV's because I was having trouble with access privileges to my google drive doc.

![Capture](https://user-images.githubusercontent.com/25185661/83409617-c0c08400-a3e2-11ea-8b67-b30025fdb1f1.JPG)
![Capture2](https://user-images.githubusercontent.com/25185661/83409618-c1591a80-a3e2-11ea-9333-f73559f74590.JPG)

**Comment by nstrayer** [comment]
Figured it out! 

In `cv_printing_functions.R`, find the line: 
```
description_bullets = paste0("- ", description_bullets),
```
and replace it with 
```
description_bullets = ifelse(description_bullets != "", paste0("- ", description_bullets), ""),
```

I was adding a bullet before the descriptions because the first description didn't get a bullet appended with the collapsing using `paste(collapse = "-")` that I did to merge. I neglected to check if there were any bullets at all though. 

This will be updated in the package as well. Thanks for pointing it out!

**Comment by nstrayer** [comment]
I just realized that @will-ball's issue might not be solved by my previous work. 
@will-ball , do you have any entries with a description 3 in them?

**Comment by nstrayer** [comment]
May have also been fixed by #25 where I made all columns be loaded as a character string which may have fixed an issue with google sheets not knowing what to do with an empty column

**Comment by MLiberati** [comment]
The "description_bullets" edit fixed it for me! Thanks!

**Comment by will-ball** [comment]
> May have also been fixed by #25 where I made all columns be loaded as a character string which may have fixed an issue with google sheets not knowing what to do with an empty column

Hi @nstrayer, Thanks for your efforts. Replacing the 'description_bullets' hasn't fixed the issue for me. In one of the versions I had something in a description_3 box but don't have any longer.

I'm intrigued by the #25 fix - is there a quick way to update to that version of the files without losing any information? Should I just run use_datadrive_cv() again?

Thanks,
Wil

**Comment by will-ball** [comment]
Hi Nick,

I've managed to get around to re-downloading the most recent version of the package and that seems to have fixed it.

Thanks,
Will
